CONTRIBUTING.rst

```diff
@@ -23,3 +23,53 @@ You can download the CLAs here:
 
 If you are an IBMer, please contact us directly as the contribution process is
 slightly different.
+
+======================
+Development Quickstart
+======================
+
+Clone the repo into a folder, set up a `virtual environment <https://virtualenv.pypa.io/en/latest/>`_, 
+install the requirements:
+
+.. code-block:: bash
+
+    $ git clone git clone git@github.com:cloudant/python-cloudant.git
+    $ cd python-cloudant
+    $ virtualenv .
+    $ ./bin/activate
+    $ pip install -r requirements.txt
+    $ pip install -r test-requirements.txt
+    
+Before running the tests, start CouchDB:
+
+.. code-block:: bash
+    
+    $ couchdb
+
+The tests create databases in your CouchDB instance, these are `db-<uuid4()>`. 
+They also create and delete documents in the `_replicator` database.
+
+Now, run the tests. Here, I use the ``ADMIN_PARTY`` environment variable to
+tell the tests not to use any authentication. See below for the full set of
+variables that can be used.
+
+.. code-block:: bash
+
+    $ ADMIN_PARTY=true nosetests -w ./tests/unit
+    
+There are several environment variables which affect
+test behaviour:
+
+- ``RUN_CLOUDANT_TESTS``: set this to run the tests that use Cloudant-specific features.
+  If you use this, you must set ``DB_URL``, ``DB_USER`` and ``DB_PASSWORD``.
```

Comment:
`DB_USER` and `DB_PASSWORD` must be set and either `DB_URL` or `CLOUDING_ACCOUNT`.  So you can set either `DB_USER`, `DB_PASSWORD`, `DB_URL` or `DB_USER`, `DB_PASSWORD`, `CLOUDING_ACCOUNT` to run tests against Cloudant.